src/test/run-make/verbatim/Makefile

```diff
@@ -0,0 +1,24 @@
+-include ../../run-make-fulldeps/tools.mk
```

Comment:
Could you rename the test directory name from just `verbatim` to `native-link-modifier-verbatim`?